spotifyPlayer.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, spotify_client_id, spotify_client_secret):
 
 		self.currSongID = 1
 		self.mp3Files = []
-		self.mixer = mixer.init()
+		mixer.init()
 
 		self.tempDir = "mp3FilesTemp"
```

Comment:
Don't manually manage the creation and location of temporary files. Use [the built-in library](https://docs.python.org/3/library/tempfile.html).